build_tools/wheels/build_wheels.sh

```diff
@@ -59,6 +59,13 @@ if [[ "$GITHUB_EVENT_NAME" == "schedule" || "$CIRRUS_CRON" == "nightly" ]]; then
     export CIBW_BUILD_FRONTEND='pip; args: --pre --extra-index-url "https://pypi.anaconda.org/scientific-python-nightly-wheels/simple"'
 fi
 
+if [[ "$CIBW_FREE_THREADED_SUPPORT" =~ [tT]rue ]]; then
```

Comment:
The regex here is because I am not too sure about yaml value plus its env var translation. It looks like you can use `true` and `True` in the yaml and the env variable will be `"true"`. You could imagine using `"True"` in the yaml and that should still work I think.